styles.css

```diff
@@ -24,14 +37,12 @@ nav {
 	flex-direction: row;
```

Comment:
I would suggest using gap instead of width 30%. This would have the benefit of not squishing the items together on mobile as well as holding proportions as the screen changes size.